source/reference/minio-mc/mc-cp.rst

```diff
@@ -58,7 +58,7 @@ similar results to the ``cp`` commandline tool.
                           [--attr "string"]                                         \
                           [--continue]                                              \
                           [--disable-multipart]                                     \
-                          [--encrypt "string"]                                      \
+                          [--encrytp "string"]                                      \
```

Comment:
This was unintentional, I'd imagine.

```suggestion
                          [--encrypt "string"]                                      \
```

Reply:
Apparently I used the emacs `mangle-words` command by accident. Fixed literally seconds before your comment.